server/calendar/helpers.js

```diff
@@ -2,63 +2,76 @@
 
 const url = require('url');
 const entities = require('entities');
+const sanitizeHtml = require('sanitize-html');
 const unquote = require('unquote');
 
 const reDate = /^\d{1,2}\/\d{1,2}\d{4}$/;
 const reUrl = /url\((.*?)\)/;
 
 const baseUrl = 'http://infozona.hr';
 
+const textFilter = text => text.replace(/\n/g, '<br>');
+
+const sanitizeHTML = html => sanitizeHtml(html, textFilter);
+
 module.exports = {
-  readDate, readTime,
-  readInfo, readPhotoUrl, readLink
+  readDate, readTime, readInfo,
+  readPhotoUrl, readLink, sanitizeHTML
 };
 
 function readDate($date) {
   let date = $date.text().trim();
-  if (reDate.test(date)) return date;
-  return formatDate(new Date());
+  return reDate.test(date) ? normalizeHrDateString(date) : formatDate(new Date());
 }
 
 function readTime($time) {
-  let [ time ] = $time.text().trim().split(/\s+/);
+  const [ time ] = $time.text().trim().split(/\s+/);
   return time.replace(/\./, ':');
 }
 
 function readInfo($info) {
-  let html = entities.decode($info.html());
-  let records = html.trim().split('<br>');
-  records = records.filter(r => r.length > 0);
-
-  let info = {};
-  records.forEach(record => {
-    let [key, val] = record.split(/:\s*/);
-    info[key] = val;
-  });
-  return info;
+  const html = entities.decode($info.html());
+  return html
+    .trim()
+    .split('<br>')
+    .filter(r => r.length > 0)
+    .reduce((info, record) => {
+      const [key, val] = record.split(/:\s*/);
+      info[key] = val;
+      return info
+    }, {});
```

Comment:
Well, technically you could turn it all into one big chain 🚂 🚋 🚋 🚋  😁

Reply:
![Train](http://weknowmemes.com/wp-content/uploads/2012/10/go-home-bus-youre-drunk.jpg)